src/components/cards/ClientQuoteCard.tsx

```diff
@@ -2,35 +2,32 @@ import type { ClientQuoteCardProps } from '@/interfaces/Card/ClientQuoteCardInte
 import AddressFormat, { DateFormat, timeAgoFormat } from '@/utils/Format';
 import MovingTypeChips from '../chips/MovingTypeChips';
 
-export default function ClientQuoteCard({ data }: ClientQuoteCardProps) {
-  // Api 연결 필요
-  // let status: string = 'end';
-  // let status: string = 'abandon';
-
+export default function ClientQuoteCard({ data, owner, designatedRequest }: ClientQuoteCardProps) {
   return (
     <>
       <div className="relative w-full rounded-[1.6rem] border border-line-100 lg:pt-[2rem] lg:pb-[1.2rem] lg:px-[2.4rem] md:p-[1.6rem] sm:py-[1.6rem] sm:px-[1.4rem] shadow-custom9">
         <div className="flex flex-col gap-[1.6rem]">
           <div className="flex justify-between items-center">
             <div className="flex lg:gap-[1.2rem] sm:gap-[0.8rem]">
-              {/* 견적 확정시에만 RECEIVED 띄우기 */}
-              <MovingTypeChips type="RECEIVED" />
-              <MovingTypeChips type={data.moveInfo.type} />
+              {data.progress === 'OPEN' && <MovingTypeChips type="WAITING" />}
+              {(data.progress === 'CONFIRMED' || 'COMPLETE') && <MovingTypeChips type="RECEIVED" />}
```

Comment:
```suggestion
              {(data.progress === 'CONFIRMED' || data.progress === 'COMPLETE') && <MovingTypeChips type="RECEIVED" />}
```
이렇게 작성하지 않아도 두 가지 경우 모두 잘 적용되나요??
같은 이슈를 겪었던 적이 있어 궁금합니다!

Reply:
아 감사합니다 ! 아래 처럼 수정해놓겠습니다 !